setup.py

```diff
@@ -1,21 +1,17 @@
-#!/usr/bin/env python
+# Copyright (C) 2023  Dimitios Mylonas
 
-# Copyright (C) 2009, Simon Schampijer
-#
```

Comment:
You removed the original author when his code is still part of this file and you haven't removed everything he wrote.

Reply:
I wanted to maintain the same author throughout the files. And since I changed almost everything and the setup file is the default used for every activity, I didn't see a reason for keeping the original author's copyright for it.

Reply:
The best reason for keeping the original author's copyright  on an otherwise unchanged file is to avoid fraudulent dealing with copyright, which is an offence socially in some online cultures, and legally in some countries.  Please make sure you understand copyright law and have fully read and understood the license you are using.